R/build-home-index.R

```diff
@@ -74,7 +133,7 @@ sidebar_section <- function(heading, bullets, class = make_slug(heading)) {
     return(character())
 
   paste0(
-    "<div class='", class, "'>\n",
+    "<div id='sidebar-", class ,"'class='", class, "'>\n",
```

Comment:
Not sure this is needed but I was thinking about https://github.com/r-lib/pkgdown/issues/1181

Reply:
Missing a space?